lib/pages/media_view.dart

```diff
@@ -11,6 +13,8 @@ import '../widgets/bottom_modal.dart';
 class MediaViewPage extends HookWidget {
   final String url;
   final GlobalKey<ScaffoldState> _key = GlobalKey();
+  static const dismissThreshold = 150;
+  static const velocityThreshold = 1000;
```

Comment:
Well technically both are dismiss thresholds, but one is for speed (not velocity) and the other one is for distance

Reply:
>speed (not velocity)

i think not
![image](https://user-images.githubusercontent.com/10037914/107086865-df410f80-67fa-11eb-8edb-75a5bf6bca7b.png)